src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -1285,6 +1290,11 @@ private void CalculateIORedirection(out bool redirectOutput, out bool redirectEr
                     redirectError = true;
                 }
             }
+            else if (_isTranscribing && (false == s_supportScreenScrape))
```

Comment:
Maybe remove the `else`? The `else if (Platform.IsWindowsDesktop && IsConsoleApplication(this.Path))` block doesn't necessarily set `redirectOutput` and `redirectError` to true.

Reply:
And please add a comment that explains why we need to set output/error redirection in such case.

Reply:
yup